lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -1116,12 +1116,15 @@ namespace Js
         AddFunctionToLibraryObject(globalObject, PropertyIds::escape, &GlobalObject::EntryInfo::Escape, 1);
         AddFunctionToLibraryObject(globalObject, PropertyIds::unescape, &GlobalObject::EntryInfo::UnEscape, 1);
 
-        if (scriptContext->GetConfig()->SupportsCollectGarbage()
```

Comment:
There is only one consumer of SupportsCollectGarbage remaining under ENABLE_SCRIPT_PROFILING ifdef. Given the API always returns "true", is it worth removing it altogether?